skimage/transform/_geometric.py

```diff
@@ -870,6 +879,20 @@ class AffineTransform(ProjectiveTransform):
     ------
     ValueError
         If both ``matrix`` and any of the other parameters are provided.
+
+    References
+    ----------
+    .. [1] Wikipedia, "Image transformation" section of "Affine transformation":
+           <https://en.wikipedia.org/wiki/Affine_transformation>
+    .. [2] Wikipedia, :"Shear mapping"
+           <https://en.wikipedia.org/wiki/Shear_mapping>
```

Comment:
The formatting is inconsistent here. How about
```suggestion
    .. [1] Wikipedia, "Affine transformation",
           https://en.wikipedia.org/wiki/Affine_transformation#Image_transformation
    .. [2] Wikipedia, "Shear mapping",
           https://en.wikipedia.org/wiki/Shear_mapping
```

Reply:
Thanks. You are right.